src/main/java/io/jenkins/plugins/gitlabserver/servers/GitLabServer.java

```diff
@@ -139,6 +135,26 @@ public String getCredentialsId() {
         return credentialsId;
     }
 
+    /**
```

Comment:
Is there a reason this constructor is moving around? It moved in #4 also. :)

Reply:
Yeah this is a silly reason actually, I just felt like it should be below all getters as it will be easy to look up at the `DescriptorImpl`.

Reply:
No worries. I'm guessing that it's probably going to stick here. Just be mindful that your reviewers have to look through the change a bit. Not a big deal but it just adds to review time. I think your reasoning makes sense.